Makefile

```diff
@@ -174,7 +174,7 @@ postgres: cert-manager
 
 webhook: cert-manager
 	-kubectl create ns $(KIND_NAMESPACE)
-	sed 's/namespace: open-cluster-management/namespace: $(KIND_NAMESPACE)/g' deploy/webhook.yaml | kubectl apply -f-
+	sed -E 's,open-cluster-management(.svc|/|$$),$(KIND_NAMESPACE)\1,g' deploy/webhook.yaml | oc apply -f -
```

Comment:
Is there a reason this uses `oc` instead of `kubectl`?

Reply:
Habit 🤦‍♂️ 

It's fixed. 🙂